### PR TITLE
Adds useful sec officer tips

### PR DESCRIPTION
**Pull request by ghost**
i mean, this should be common sense, but whatever

:cl: More Robust Than You
add: Added some useful sec officer tips
/:cl:

### CODE REVIEW

**Comment by awesine** [comment]
please also add "You can attach your sec-light to your taser!"

**Comment by MoreRobustThanYou** [comment]
done @awesine

**Comment by awesine** [comment]
thanks it's a surprise how many people don't know that

**Comment by deathride58** [comment]
You can also attach your seclite to your helmet, I'm surprised literally nobody knows that.

**Comment by FantasticFwoosh** [comment]
> You can also attach your seclite to your helmet

Pretty standard, its actually better than attaching to your gun because if you are disarmed you'd be left in the dark, wheras having it attached to your armor means they have to undress you first.

**Comment by Anonmare** [comment]
I used to put a seclite on my helmet, my taser and keep one in my pocket to be a shining light in the darkness

File: strings/tips.txt [hunk 1]
```diff
@@ -186,3 +186,4 @@ As a Ghost, you can double click on people, bots, or the singularity to follow t
 As a Devil, you gain power for every three souls you control, however you also become more obvious.
 As a Devil, as long as you control at least one other soul, you will automatically resurrect, as long as a banishment ritual is not performed.
 At which time a Devil's nameth is spake on the tongue of man, the Devil may appeareth. 
+As a Security Officer, remember that causation does not equal correlation. Someone may have just been at the wrong place at the wrong time!
```
Comment (Supermichael777) [inline strings/tips.txt]: other way around, causation is a correlation but not all correlations imply causation

**Review by Supermichael777 — CHANGES_REQUESTED** [review]

**Comment by StreakyHaddock** [comment]
@deathride58 WHAT.

**Review by Cyberboss — DISMISSED** [review]

**Review by Cyberboss — DISMISSED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**